packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -568,6 +568,8 @@
   "Enable_Svg_Favicon": "Enable SVG favicon",
   "Enable_two-factor_authentication": "Enable two-factor authentication",
   "Enabled": "Enabled",
+  "Enable_Auto_Away": "Enable Auto Away",
+  "Enable_Svg_Favicon": "Enable SVG favicon",
```

Comment:
Is this string addition correct?

Reply:
I believe that the "Enable Svg Favicon" came from a merge made to resolve some conflicts. I may need to look at this PR again now :/